sound/soc/sof/core.c

```diff
@@ -340,9 +340,17 @@ EXPORT_SYMBOL(snd_sof_device_probe);
 
 int snd_sof_device_remove(struct device *dev)
```

Comment:
@mrajwa And the second patch has lines wrapped to 60. :) 

Btw, the "due to HW bug related to LPSRAM registers" comment is a bit out of place as you are changing code that is used for non-Intel hardware as well and this does not specify which hardware. So maybe better to put:

""
On some Intel hardware, DSP has to power gate all LPSRAM banks at
power-down or otherwise the LPSRAM is not accessible upon following
power-up. 
""

Reply:
Fair point I will update